### PR TITLE
Create LIT6945CommentHebrewWRuf.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
A new work record. the work is a mixture of quotations and re-phrasings from the Commentary of Chsysostom intermixed with Amharic commentaries, written from dictation.

### CODE REVIEW

File: new/LIT6945CommentHebrewWRuf.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Explications to the Gǝʿǝz <ref type="work" corresp="#LIT3171Commen">Commentary on
```
Comment (eu-genia) [inline new/LIT6945CommentHebrewWRuf.xml:36]: delete `#`

File: new/LIT6945CommentHebrewWRuf.xml [hunk 2]
```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6945CommentHebrewWRuf" xml:lang="amh" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Explications by Walda Rufāel of Goǧǧām in Amharic on 
+                    Commentary on the Epistle to the Hebrews by John Chrysostom</title>
```
Comment (eu-genia) [inline new/LIT6945CommentHebrewWRuf.xml:9]: **the** Commentary

File: new/LIT6945CommentHebrewWRuf.xml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        <revisionDesc>
+            <change who="DN" when="2024-03-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="dcterms:creator" active="LIT6945CommentHebrewWRuf" passive="PRS14363WaldaRufael"/>
+                    <relation name="iha:Commentary_of" active="LIT6945CommentHebrewWRuf" passive="LIT3171Commen"/>
```
Comment (eu-genia) [inline new/LIT6945CommentHebrewWRuf.xml:62]: never use `iha `prefixed keywords, relations, files etc, these are IHA-project specific, we use `saws `(saws:isCommentOn)

File: new/LIT6945CommentHebrewWRuf.xml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Explications to the Gǝʿǝz <ref type="work" corresp="#LIT3171Commen">Commentary on 
+                    the Epistle to the Hebrews by John Chrysostom</ref> dictated by <persName ref="PRS14363WaldaRufael">ʾalaqā Walda Rufāʾel of Goǧǧām</persName> 
+                    at <placeName ref="LOC6373Yawes">Yawǝš</placeName>, upon the order of <persName ref="PRS4779GossuZa"/>. According to A. d'Abbadie, 
+                    a copy was made from the <persName ref="PRS4779GossuZa">Goššu's</persName> own manuscript.</p>
```
Comment (eu-genia) [inline new/LIT6945CommentHebrewWRuf.xml:39]: d'Abbadie in bibliography?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6945CommentHebrewWRuf.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/LIT6945CommentHebrewWRuf.xml:36]: ok

File: new/LIT6945CommentHebrewWRuf.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LIT6945CommentHebrewWRuf.xml:9]: ok

File: new/LIT6945CommentHebrewWRuf.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/LIT6945CommentHebrewWRuf.xml:62]: ok

File: new/LIT6945CommentHebrewWRuf.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply new/LIT6945CommentHebrewWRuf.xml:39]: done

**Review by eu-genia — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**